teku/src/main/java/tech/pegasys/teku/AbstractNode.java

```diff
@@ -59,7 +60,18 @@ public abstract class AbstractNode implements Node {
   protected final ServiceConfig serviceConfig;
 
   protected AbstractNode(final TekuConfiguration tekuConfig) {
+    final String network =
+        tekuConfig
+            .eth2NetworkConfiguration()
+            .getEth2Network()
+            .map(Eth2Network::configName)
+            .orElse("empty");
+    final String storageMode = tekuConfig.storageConfiguration().getDataStorageMode().name();
+    final int restApiPort = tekuConfig.beaconChain().beaconRestApiConfig().getRestApiPort();
+
     STATUS_LOG.onStartup(VersionProvider.VERSION);
+    STATUS_LOG.startupConfigurations(network, storageMode, restApiPort);
```

Comment:
I believe we should encapsulate the logic of what information goes into the log in a separate object (e.g. `StartupConfigLog`). This will give us more flexibility in testing and future changes. Might even rely on a `toString()` method to actually build the message if we decide to do it all as one string.

This way, you can update StatusLogger to have a method like `StatusLogger.startupConfiguration(StartupLogConfig config)` that we don't have to change the signature every time we want to add/remove/change the startup config.

I am not too sure about the value of having this part of StatusLogger, but if there is a reason to have them tied up, we can invert the dependency and have `StartupConfigLog` receive a Consumer<StartupConfigLog> in its constructor that will be executed when you call a `report()` method. And have the consumer be provided by StatusLogger.